### PR TITLE
lurk: add aarch64-linux support

### PR DESCRIPTION
**Pull request by gepbird**
From a previous release https://github.com/JakWai01/lurk/releases/tag/v0.3.7:
- Lurk now has Aarch64 support 🚀

I have not added darwin as most likely it still [doesn't work](https://github.com/JakWai01/lurk/issues/4#issuecomment-2354536756) and I can't test it.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by getchoo** [comment]
@ofborg build lurk

File: pkgs/by-name/lu/lurk/package.nix [hunk 1]
```diff
@@ -10,8 +10,8 @@ rustPlatform.buildRustPackage rec {
 
   src = fetchFromGitHub {
     owner = "jakwai01";
-    repo = pname;
-    rev = "v${version}";
+    repo = "lurk";
+    rev = "refs/tags/v${version}";
```
Comment (getchoo) [inline pkgs/by-name/lu/lurk/package.nix:14]:
```suggestion
    tag = "v${version}";
```
Nit. See https://github.com/NixOS/nixpkgs/pull/355973

**Review by getchoo — APPROVED** [review]

**Review by getchoo — APPROVED** [review]

File: pkgs/by-name/lu/lurk/package.nix [hunk 2]
```diff
@@ -22,16 +22,19 @@ rustPlatform.buildRustPackage rec {
       --replace-fail '/usr/bin/ls' 'ls'
   '';
 
-  meta = with lib; {
+  meta = {
+    changelog = "https://github.com/jakwai01/lurk/releases/tag/${src.rev}";
```
Comment (getchoo) [inline pkgs/by-name/lu/lurk/package.nix:26]:
```suggestion
    changelog = "https://github.com/jakwai01/lurk/releases/tag/v${version}";
```
Moving to `tag` unset `rev`. You can re-use version here or `src.tag` again, up to you

**Review by getchoo — CHANGES_REQUESTED** [review]

File: pkgs/by-name/lu/lurk/package.nix (hunk 2 above)
Comment (gepbird) [reply pkgs/by-name/lu/lurk/package.nix]: Yeah, just got notified of the eval failure, I was surprised it wasn't using version before, thanks anyway!

**Review by getchoo — APPROVED** [review]

**Pull request merged**